### PR TITLE
Add testing of  alma API

### PR DESCRIPTION
**Pull request by dr0i**
See #1686.

### CODE REVIEW

File: web/app/controllers/resources/LocalIndex.java [hunk 1]
```diff
@@ -38,27 +38,30 @@
...
 public class LocalIndex {
 
-	private static final String TEST_CONFIG =
+	private static String TEST_CONFIG =
 			"../src/main/resources/index-config.json";
 	private static final String TEST_DATA = "../src/test/resources/jsonld";
 	private static final String TEST_DATA_ITEMS = TEST_DATA + "/items";
 
-	private Node node;
+    private Node node;
```
Comment (fsteeg) [inline web/app/controllers/resources/LocalIndex.java:52]: Formatting seems off here, and in some other places below.

File: web/test/tests/almafix/LocalIndexSetup.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/**
+ * Setup for the alma fix search tests. Creates a local ES index with test data.
+ *
+ * @author Fabian Steeg (fsteeg)
+ */
+@SuppressWarnings("javadoc")
+public abstract class LocalIndexSetup {
+
+    private static LocalIndex index;
+    private static String TEST_CONFIG =
```
Comment (fsteeg) [inline web/test/tests/almafix/LocalIndexSetup.java:20]: This could be `final`, right? If not, as above, it should  be called `testConfig`.

**Review by fsteeg — COMMENTED** [review]
Commented on some minor issues inline.

And a general question: the `almafix` package duplicates the original tests? Isn't the API basically the same? I would have expected something like a switch for setting where the data comes from (morph/aleph or fix/alma), but the tests remaining the same. I'm probably missing something here.

File: web/test/tests/almafix/LocalIndexSetup.java (hunk 2 above)
Comment (dr0i) [reply web/test/tests/almafix/LocalIndexSetup.java]: right. fixed that.

File: web/test/tests/IndexIntegrationTest.java [hunk 3]
```diff
@@ -98,7 +98,6 @@ private static List<Object[]> queries(Object[][] objects) {
 			String s = (String) testCase[0];
 			Integer hits = (Integer) testCase[1];
 			result.add(new Object[] { new Queries.Builder().q(s), /*->*/ hits });
-			result.add(new Object[] { new Queries.Builder().word(s), /*->*/ hits });
```
[the remark this answers is not in the document]
Comment (dr0i) [reply web/test/tests/IndexIntegrationTest.java:101]: Like the commit describes -it's redundant. It just tests the exact input/output all over again.  Maybe I missed something.

**Review by fsteeg — APPROVED** [review]

File: web/test/tests/IndexIntegrationTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (fsteeg) [reply web/test/tests/IndexIntegrationTest.java:101]: Yes, I guess it's alright, we should probably come up with a test in the future that actually shows how these methods return different results.

**Pull request merged**